src/pages/dev/amplifier/chain-integration/relay-messages/automatic.mdx

```diff
@@ -0,0 +1,33 @@
+# Automatically relay messages with the Amplifier API
+
+import { Callout } from "/src/components/callout"
+
+<Callout>
+If you wish to relay messages manually, check out [Manually relay messages through Amplifier](/dev/amplifier/chain-integration/relay-messages/).
+</Callout>
+
+Chains connected through the Amplifier must be able to send and receive GMP data with the Axelar Virtual Machine (AVM). The Amplifier API relayer is the component responsible for delivering this data. It does this through two distinct workflows:
```

Comment:
> Chains connected through the Amplifier must be able to send and receive GMP data

Is this the wording we usually go with in the docs? It's technically not "the connected chains" that send and receive GMP data (in my head that implies that the chain would need to support it natively in the consensus layer), it's rather us/partners that are supporting the chain by implementing the workflows you mention below.

Except if you mean that chains should be EVM compatible (i.e., support smart contracts), in which case we could phrase it more directly.